app/src/main/java/com/teamfillin/fillin/HomeActivity.kt

```diff
@@ -0,0 +1,60 @@
+package com.teamfillin.fillin
+
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import android.view.View
+import android.widget.Toast
+import androidx.databinding.DataBindingUtil.setContentView
+import com.teamfillin.fillin.databinding.ActivityHomeBinding
+import kotlinx.coroutines.newFixedThreadPoolContext
+
+class HomeActivity : AppCompatActivity() {
+    private lateinit var binding: ActivityHomeBinding
+    private lateinit var newPhotosAdapter: NewPhotosAdapter
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+
+        binding = ActivityHomeBinding.inflate(layoutInflater)
+        initAdapter()
+        popup()
+        setContentView(binding.root)
+
+    }
+
+
+    private fun initAdapter() {
+        newPhotosAdapter = NewPhotosAdapter()
+
+        binding.rcvNewPhotos.adapter = newPhotosAdapter
+
+        val exphoto = R.drawable.and_photo_rectangle
+
+        newPhotosAdapter.photolist.addAll(
+            listOf(
+                NewPhotosData(exphoto),
+                NewPhotosData(exphoto),
+                NewPhotosData(exphoto),
+                NewPhotosData(exphoto),
+                NewPhotosData(exphoto),
+                NewPhotosData(exphoto),
+                NewPhotosData(exphoto),
+                NewPhotosData(exphoto)
+            )
+        )
+        newPhotosAdapter.notifyDataSetChanged()
+    }
+
+    fun popup() {
+        binding.apply{
+            btnClose.setOnClickListener {
+                if (clPopup.getVisibility() == View.VISIBLE)
+                    clPopup.setVisibility(View.GONE);
+
+            }
+                tvNotice.setOnClickListener {
+                    Toast.makeText(this@HomeActivity, "현상소 제보 Page이동", Toast.LENGTH_SHORT).show()
+                }
+        }
```

Comment:
- 줄 정렬 안되어있습니다
- getVisibility, setVisibility는 java에서 사용하는 방식입니다. 코틀린에서는 visibility라는 프로퍼티를 통해서 접근할 수 있습니다.
- visibility를 활용하는 것보단 isVisible을 활용하여 boolean 값으로 visibility를 조절할 수 있습니다. 이걸 활용해보는 건 어떨까요?

```suggestion
                // core 모듈 내에 있는 기능 ㅎㅎ
                tvNotice.setOnClickListener {
                    toast("현상소 제보 Page이동")
                }
```

- apply는 블록안에 있는 사항들을 적용시킨 객체를 다른 변수에 대입할 때 사용하는 스코프 함수입니다. with(binding)이 이런 경우에서 일반적으로 사용됩니다.